spec/jobs/pusher_job_spec.rb

```diff
@@ -3,8 +3,12 @@
 describe PusherJob, :type => :job do
 
   it "should call twilio with options" do
-    expect(Pusher).to receive(:trigger).with(['channel1', 'channel2'], 'update', {one: 1})
-    PusherJob.new.perform(['channel1', 'channel2'], 'update', {one: 1})
+    channels = ['channel1', 'channel2']
+    event = 'update'
+    data = {one: 1}.to_json
+
+    expect(Nestful).to receive(:post).with(any_args, {rooms:channels, event:event, args:JSON.parse(data)}, any_args)
```

Comment:
Line is too long. [116/80]<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.